{
   "active_lock_reason" : null,
   "assignee" : null,
   "assignees" : [],
   "author_association" : "MEMBER",
   "body" : "What looks like an intermittent failure in `feature_config_args.py` happened in CI testing of #21407.\r\n\r\nhttps://cirrus-ci.com/task/5952013763608576\r\n\r\n```\r\n test  2021-03-16T11:11:46.877000Z TestFramework (ERROR): Assertion failed \r\n                                   Traceback (most recent call last):\r\n                                     File \"/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_framework.py\", line 128, in main\r\n                                       self.run_test()\r\n                                     File \"/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/feature_config_args.py\", line 219, in run_test\r\n                                       self.test_seed_peers()\r\n                                     File \"/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/feature_config_args.py\", line 214, in test_seed_peers\r\n                                       self.nodes[0].setmocktime(start + 65)\r\n                                     File \"/usr/lib/python3.6/contextlib.py\", line 88, in __exit__\r\n                                       next(self.gen)\r\n                                     File \"/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_node.py\", line 400, in assert_debug_log\r\n                                       self._raise_assertion_error('Expected messages \"{}\" does not partially match log:\\n\\n{}\\n\\n'.format(str(expected_msgs), print_log))\r\n                                     File \"/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_node.py\", line 166, in _raise_assertion_error\r\n                                       raise AssertionError(self._node_msg(msg))\r\n                                   AssertionError: [node 0] Expected messages \"['Adding fixed seeds as 60 seconds have passed and addrman is empty']\" does not partially match log:\r\n                                    - 2021-03-16T11:10:26.849988Z (mocktime: 2021-03-16T11:10:26Z) [httpserver.cpp:238] [http_request_cb] Received a POST request for / from 127.0.0.1:35268\r\n                                    - 2021-03-16T11:10:26.850095Z (mocktime: 2021-03-16T11:10:26Z) [rpc/request.cpp:174] [parse] ThreadRPCServer method=setmocktime user=__cookie__\r\n                                    - 2021-03-16T11:10:26.853443Z (mocktime: 2021-03-16T11:11:31Z) [util/system.h:468] [TraceThread] msghand thread start\r\n                                    - 2021-03-16T11:10:26.853489Z (mocktime: 2021-03-16T11:11:31Z) [util/system.h:468] [TraceThread] opencon thread start\r\n                                    - 2021-03-16T11:10:26.853541Z (mocktime: 2021-03-16T11:11:31Z) [util/system.h:468] [TraceThread] addcon thread start\r\n                                    - 2021-03-16T11:10:26.853580Z (mocktime: 2021-03-16T11:11:31Z) [net.cpp:394] [ConnectNode] trying connection fakenodeaddr lastseen=0.0hrs\r\n                                    - 2021-03-16T11:10:26.854222Z (mocktime: 2021-03-16T11:11:31Z) [util/system.h:468] [TraceThread] net thread start\r\n                                    - 2021-03-16T11:11:26.797916Z (mocktime: 2021-03-16T11:11:31Z) [random.cpp:523] [SeedPeriodic] Feeding 14413 bytes of dynamic environment data into RNG\r\n                                    - 2021-03-16T11:11:27.362393Z (mocktime: 2021-03-16T11:11:31Z) [net.cpp:394] [ConnectNode] trying connection fakenodeaddr lastseen=0.0hrs\r\n```\r\nIt  is expecting the messsage \"Adding fixed seeds as 60 seconds have passed and addrman is empty\" but is not getting it.",
   "closed_at" : "2021-05-05T14:48:29Z",
   "closed_by" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/6399679?v=4",
      "events_url" : "https://api.github.com/users/MarcoFalke/events{/privacy}",
      "followers_url" : "https://api.github.com/users/MarcoFalke/followers",
      "following_url" : "https://api.github.com/users/MarcoFalke/following{/other_user}",
      "gists_url" : "https://api.github.com/users/MarcoFalke/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/MarcoFalke",
      "id" : 6399679,
      "login" : "MarcoFalke",
      "node_id" : "MDQ6VXNlcjYzOTk2Nzk=",
      "organizations_url" : "https://api.github.com/users/MarcoFalke/orgs",
      "received_events_url" : "https://api.github.com/users/MarcoFalke/received_events",
      "repos_url" : "https://api.github.com/users/MarcoFalke/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/MarcoFalke/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/MarcoFalke/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/MarcoFalke"
   },
   "comments" : 2,
   "comments_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/21448/comments",
   "created_at" : "2021-03-16T11:37:49Z",
   "events_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/21448/events",
   "html_url" : "https://github.com/bitcoin/bitcoin/issues/21448",
   "id" : 832703331,
   "labels" : [
      {
         "color" : "d4c5f9",
         "default" : false,
         "description" : null,
         "id" : 62963516,
         "name" : "Tests",
         "node_id" : "MDU6TGFiZWw2Mjk2MzUxNg==",
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/labels/Tests"
      }
   ],
   "labels_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/21448/labels{/name}",
   "locked" : false,
   "milestone" : null,
   "node_id" : "MDU6SXNzdWU4MzI3MDMzMzE=",
   "number" : 21448,
   "performed_via_github_app" : null,
   "repository_url" : "https://api.github.com/repos/bitcoin/bitcoin",
   "state" : "closed",
   "title" : "test: Intermittent failure in feature_config_args",
   "updated_at" : "2021-05-05T14:48:29Z",
   "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/21448",
   "user" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/126646?v=4",
      "events_url" : "https://api.github.com/users/laanwj/events{/privacy}",
      "followers_url" : "https://api.github.com/users/laanwj/followers",
      "following_url" : "https://api.github.com/users/laanwj/following{/other_user}",
      "gists_url" : "https://api.github.com/users/laanwj/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/laanwj",
      "id" : 126646,
      "login" : "laanwj",
      "node_id" : "MDQ6VXNlcjEyNjY0Ng==",
      "organizations_url" : "https://api.github.com/users/laanwj/orgs",
      "received_events_url" : "https://api.github.com/users/laanwj/received_events",
      "repos_url" : "https://api.github.com/users/laanwj/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/laanwj/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/laanwj/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/laanwj"
   }
}
